code/Evaluation_flu_hosp.Rmd

```diff
@@ -149,6 +159,7 @@ diff_weeks_season = as.numeric(diff_weeks_season)
 
 ```{r load data}
 #load files generated in query-scores-weekly-report.R
+# setwd("~/github/flusight-eval/reports")

```

Comment:
If you're going to leave this commented-out line for development, let's put something like:

# for dev running locally outside of container: uncomment the following line